content/spire/try/oidc-federation-aws.md

```diff
@@ -0,0 +1,415 @@
+---
+title: AWS OIDC Authentication
+description: Using OIDC to Authenticate SPIRE to AWS S3 on Kubernetes
+weight: 4
+toc: true
+aliases:
+menu:
+  spire:
+    weight: 50
+    parent: 'spire-try'
+---
+
+This tutorial builds on the [Kubernetes Quickstart](/spire/try/getting-started-k8s/) guide to describe how a SPIRE identified workload can, using a JWT-SVID, authenticate to Amazon AWS APIs, assume an AWS IAM role, and retrieve data from an AWS S3 bucket. This avoids the need to create and deploy AWS IAM credentials with the workload itself.
+
+In this tutorial you will learn how to:
+
+* Deploy the OIDC Discovery Provider Service
+* Create the required DNS A record to point to the Discovery Provider document
+* Create a sample AWS identity provider, policy, role, and S3 bucket
+* Test access to the S3 bucket
+
+## Prerequisites
+
+Note the following required accounts, prerequisites, and limitations before starting this tutorial:
+
+* You'll need access to the Kubernetes environment that you configured when going through [Kubernetes Quickstart](/spire/try/getting-started-k8s/) 
+* You'll need access to the AWS console to configure an identity provider, policy, role, and S3 bucket
```

Comment:
More concretely, you'll need an account the AWS console that has the permissions to create new identity proivders, IAM policies, IAM roles and S3 buckets.

Reply:
Done, clarified AWS permissions requirement.